# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/events/test/pointerevents/mochitest.toml
# Commit: 5718f7f3067e
# Full Hash: 5718f7f3067eff42e097a6013acd82fbe6aa299a
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2025-05-15 22:58:00
# Regressor Bug: 1550462
# File Overlap Count: 1
# Description:
#   Bug 1550462 - part 2: Make `PresShell::HandleEvent` dispatch preceding `pointerrawupdate` event r=smaug,dom-core,edgar
#   
#   This patch tries to dispatch `ePointerRawUpdate` with
#   `PresShell::EventHandler::DispatchPrecedingPointerEvent` as same as usual
#   pointer events.
# ==============================================================================

diff -r aae164cff6aa -r 5718f7f3067e dom/events/test/pointerevents/mochitest.toml
--- a/dom/events/test/pointerevents/mochitest.toml	Thu May 15 03:20:28 2025 +0000
+++ b/dom/events/test/pointerevents/mochitest.toml	Thu May 15 03:20:29 2025 +0000
@@ -100,6 +100,20 @@
 
 ["test_pointermove_isPrimary_subsequent_pens.html"]
 
+["test_pointerrawupdate_event_count.html"]
+scheme = "https"
+skip-if = ["os == 'android'"]  # Bug 1312791
+
+["test_pointerrawupdate_event_count_touch.html"]
+scheme = "https"
+skip-if = [
+  "os == 'android'",  # Bug 1312791
+  "verify && os == 'win'",  # Bug 1659744
+  "win11_2009", # Bug 1781388
+  "os == 'win' && os_version == '11.26100' && processor == 'x86_64' && opt", # Bug 1781388
+]
+support-files = ["!/gfx/layers/apz/test/mochitest/apz_test_native_event_utils.js"]
+
 ["test_remove_frame_when_got_pointer_capture.html"]
 
 ["test_synthesized_touch.html"]